mac.sh

```diff
@@ -138,12 +138,12 @@ install_or_update_homebrew() {
 
 install_latest_ruby() {
   append_to_zshrc 'eval "$(rbenv init - zsh --no-rehash)"' 1
-  ruby_version="2.3.7"
+  ruby_version="2.3.7" # TODO: If we bump this, openssl should be bumped to 1.1.x, probably.
```

Comment:
https://github.com/rbenv/ruby-build/issues/1353 suggests that 2.4+ function with 1.1.1